addons/mass_mailing_sms/views/mailing_mailing_views.xml

```diff
@@ -95,6 +95,12 @@
                 <attribute name="attrs">{'invisible': ['|',('mailing_type', '!=', 'mail'),('state', 'in', ('draft','test'))]}</attribute>
             </xpath>
             <!-- Form -->
+            <xpath expr="//label[@for='subject']" position="attributes">
+                <attribute name="attrs">{'invisible': [('mailing_type', '!=', 'mail')]}</attribute>
+            </xpath>
+            <xpath expr="//label[@for='subject']" position="after">
+                <label for="sms_subject" attrs="{'invisible': [('mailing_type', '!=', 'sms')]}" />
```

Comment:
Can't we put this inside the existing group, like this:

```xml
<xpath expr="//field[@name='subject']" position="after">
    <label for="sms_subject" attrs="{'invisible': [('mailing_type', '!=', 'sms')]}" />
    <field class="text-break" name="sms_subject" string="Title" placeholder="e.g. Black Friday SMS coupon" attrs="{'invisible': [('mailing_type', '!=', 'sms')], 'readonly': [('state', 'in', ('sending', 'done'))], 'required': [('mailing_type', '=', 'sms')]}"/>
</xpath>
```

That way we simplify a bit.

Reply:
> Can't we put this inside the existing group, like this:
> 
> ```
> <xpath expr="//field[@name='subject']" position="after">
>     <label for="sms_subject" attrs="{'invisible': [('mailing_type', '!=', 'sms')]}" />
>     <field class="text-break" name="sms_subject" string="Title" placeholder="e.g. Black Friday SMS coupon" attrs="{'invisible': [('mailing_type', '!=', 'sms')], 'readonly': [('state', 'in', ('sending', 'done'))], 'required': [('mailing_type', '=', 'sms')]}"/>
> </xpath>
> ```
> 
> That way we simplify a bit.

Hello @awa-odoo 
Keeping in the existing group will misalign the things.

![image](https://github.com/odoo/odoo/assets/124355670/74044df7-0a5f-4fa3-b14f-13f69cfb1561)